backdrop/transformers/tasks/latest_transaction_explorer_values.py

```diff
@@ -0,0 +1,90 @@
+from ..worker import config
+
+from performanceplatform.client import AdminAPI
+
+from .util import encode_id, group_by, is_latest_data
+
+REQUIRED_DATA_POINTS = [
+    "cost_per_transaction",
+    "digital_cost_per_transaction",
+    "digital_takeup",
+    "number_of_digital_transactions",
+    "number_of_transactions",
+    "total_cost",
+]
+
+REQUIRED_FIELDS = [
+    "_timestamp",
+    "end_at",
+    "period",
+    "service_id",
+    "type"
+]
+
+admin_api = AdminAPI(
+    config.STAGECRAFT_URL,
+    config.STAGECRAFT_OAUTH_TOKEN)
+
+
+def _get_latest_data_points(data):
+    data.sort(key=lambda item: item['_timestamp'], reverse=True)
+    return data[0]
+
+
+def _get_stripped_down_data_for_data_point_name_only(
+        dashboard_config,
+        latest_data_points,
+        data_point_name):
+    """Builds up backdrop ready datum
+    for a single transaction explorer metric
+
```

Comment:
Sure, what specifically though? This passes my vim pep8 plugin and the docstring is pep257 compliant as far as I can work out.